src/sinks/util/buffer/metrics.rs

```diff
@@ -298,12 +325,6 @@ mod test {
             .collect()
     }
 
-    fn sorted(buffer: &[Metric]) -> Vec<Metric> {
```

Comment:
:+1: Am I following correctly that the change to the sorting here will avoid copying the metric buffer by sorting in-place?

Reply:
Correct, though that's a trivial concern given that this is only for the tests.